src/battle_tower.c

```diff
@@ -2224,11 +2224,11 @@ static void GetApprenticeMultiPartnerParty(u16 trainerId)
         }
     }
 
-    gUnknown_03006298[0] = validSpecies[Random() % count];
+    gFactoryOpponentTeam[0] = validSpecies[Random() % count];
     do
     {
-        gUnknown_03006298[1] = validSpecies[Random() % count];
-    } while (gUnknown_03006298[0] == gUnknown_03006298[1]);
+        gFactoryOpponentTeam[1] = validSpecies[Random() % count];
+    } while (gFactoryOpponentTeam[0] == gFactoryOpponentTeam[1]);
```

Comment:
`FactoryOpponent` might be misleading, considering that it's used here for a non-opponent outside the factory

Reply:
Hm, true. It’s more like the team of an NPC in various battle facilities (just mainly factory). How would you call that? `gNpcFrontierTeam`?

Reply:
I renamed it to `gFrontierNpcTeam`. Let me know if that’s better.

Reply:
What about something like `gFrontierTempParty`? In any case we tend to use "party" over "team"

Reply:
Sounds good to me.